assets/js/app/LocusZoom.js

```diff
@@ -151,5 +210,13 @@
         return response.promise;
     };
 
+    exports.isString = function(x) {
```

Comment:
I've been treating "LocusZoom" more like a namespace than a singleton. I don't want to pollute the global JS environment with functions that might conflict. For example, Angular has it's own isString. Ideally it's just be a private function not exposed but we didn't make that easy with our file layout. It would actually be easier if we only had one JS file which is something we might want to revisit. I think isString() is much easier to read and understand the intention of rather than `typeof x` and it matches better with `Array.isArray()`. We could add it to the String prototype, but again i'd like to limit changes to behave to things only within the LocusZoom namespace.